src/cmd/ksh93/tests/variables.sh

```diff
@@ -795,5 +795,118 @@ expect=1
 actual=$(env SHLVL="2#11+x[\$(env echo Exploited vuln CVE-2019-14868 >&2)0]" "$SHELL" -c 'echo $SHLVL' 2>&1)
 [[ $actual == $expect ]] || err_exit "expression allowed on env var import (expected '$expect', got '$actual')"
 
+# ======
+# Check unset behavior of special variables.
```

Comment:
This should be "Check unset and cleanup/restore behavior of special variables". Sorry, I know that's my own comment!